pkg/ceph/client/pool.go

```diff
@@ -113,7 +113,7 @@ func CreatePool(context *clusterd.Context, clusterName string, newPool CephStora
 
 func CreatePoolForApp(context *clusterd.Context, clusterName string, newPool CephStoragePoolDetails, appName string) error {
 	args := []string{"osd", "pool", "create", newPool.Name, strconv.Itoa(newPool.Number)}
-	// not implemented: fix the pool create for the different profiles
```

Comment:
how did this "not implemented" get fixed?

Reply:
this seemed like an old comment since we are creating both replicated and ec pools now. What other profiles is this referring to?

Reply:
yeah it looks like that comment came from the refactor to use external ceph tools.  as long as both EC and replicated pools are working, I think we're OK now.